### PR TITLE
[stable/clamav] Add icon for monocular

### PR DESCRIPTION
**Pull request by zakkg3**

### CODE REVIEW

File: stable/clamav/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 apiVersion: v1
-appVersion: "1.0"
+appVersion: "1.6"
```
Comment (cpanato) [inline stable/clamav/Chart.yaml:2]: why this change here? also the values.yaml is using the tag latest this is not a good practice, can you update to a version tag?

**Review by cpanato — CHANGES_REQUESTED** [review]

File: stable/clamav/Chart.yaml (hunk 1 above)
Comment (zakkg3) [reply stable/clamav/Chart.yaml:2]: my bad, it's fixed now. thanks!

**Comment by zakkg3** [comment]
@cpanato Carlos, can you review this. I know some users are waiting for this pr. Also at my office :)
thanks!!
gracias, tio. (I guess you are spanish?)

**Comment by maorfr** [comment]
/assign
/ok-to-test
/lgtm

you guessed wrong :)
